pelita/game.py

```diff
@@ -428,7 +423,10 @@ def in_homezone(position, team_id):
         'bot_positions': noised_positions['enemy_positions'],
         'is_noisy': noised_positions['is_noisy'],
         'score': game_state['score'][enemy_team],
-        'timeout_count': 0, # TODO. Could be left out for the enemy
+        'kills': game_state['kills'][enemy_team::2],
+        'deaths': game_state['deaths'][enemy_team::2],
+        'bot_eaten': game_state['bot_eaten'][enemy_team::2],
```

Comment:
Again, writing the code for `bot.eaten` on your own is not trivial, because you can be eaten while you are not playing your turn. Implementing this into the game state was really easy. This one is one I feel strong about. It should stay.

Reply:
Ok.